tests/compose/fixtures/entrypoint-composefile/docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+version: "2"
+services:
+  test:
+    image: busybox
+    entrypoint: printf
+    command: default args
```

Comment:
do you want to print `default args`? I tried this one but it only prints `default`